content/en/blog/2024/v1-apis/index.md

```diff
@@ -0,0 +1,102 @@
+
+      _There is not a strong use case for making this configurable through the Telemetry API._
+
+Please share any feedback on these fields by [creating issues on GitHub](https://github.com/istio/istio/issues).
+
+## Overview of Istio CRDs
+
+This is the full list of supported API versions:
+
+| Category | API | Versions |
+| ---------|-----|----------|
+| Networking | [Destination Rule](/docs/reference/config/networking/destination-rule/) |  `v1`, `v1beta1`, `v1alpha3` |
+| | Istio [Gateway](/docs/reference/config/networking/gateway/) |  `v1`, `v1beta1`, `v1alpha3` |
+| | [Service Entry](/docs/reference/config/networking/service-entry/) |  `v1`, `v1beta1`, `v1alpha3` |
+| | [Sidecar](/docs/reference/config/networking/sidecar/) scope |  `v1`, `v1beta1`, `v1alpha3` |
+| | [Virtual Service](/docs/reference/config/networking/virtual-service/) |  `v1`, `v1beta1`, `v1alpha3` |
+| | [Workload Entry](/docs/reference/config/networking/workload-entry/) |  `v1`, `v1beta1`, `v1alpha3` |
+| | [Workload Group](/docs/reference/config/networking/workload-group/) |  `v1`, `v1beta1`, `v1alpha3` |
+| | [Proxy Config](/docs/reference/config/networking/proxy-config/) |  `v1beta1` |
+| | [Envoy Filter](/docs/reference/config/networking/envoy-filter/) |  `v1alpha3` |
+| Security  | [Authorization Policy](/docs/reference/config/security/authorization-policy/) |  `v1`, `v1beta1` |
+| | [Peer Authentication](/docs/reference/config/security/peer_authentication/) |  `v1`, `v1beta1` |
+| | [Request Authentication](/docs/reference/config/security/request_authentication/) |  `v1`, `v1beta1` |
+| Telemetry | [Telemetry](/docs/reference/config/telemetry/) |  `v1`, `v1alpha1` |
+| Extension | [Wasm Plugin](/docs/reference/config/proxy_extensions/wasm-plugin/) |  `v1alpha1` |
+
+Istio can also be configured [using the Kubernetes Gateway API](/docs/setup/additional-setup/getting-started/).
+
+## Using the `v1` Istio APIs
+
+There are some APIs in Istio that are still under active development and are subject to potential changes between releases. For instance, the Envoy Filter, Proxy Config and Wasm Plugin APIs.
+
+Furthermore, Istio maintains a strictly identical schema across all versions of an API due to limitations in [CRD versioning](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/). Therefore, even though there is a `v1` Telemetry API, the three `v1alpha1` fields mentioned [above](#telemetry-api) can still be utilized when declaring a `v1` Telemetry API resource.
+
+For risk-averse environments, we have provided a sample `Stable Validation Policy` that can ensure only `v1` APIs and fields are used.
+
+In new environments, selecting the `Stable Validation Policy` upon installing Istio will guarantee that all future Custom Resources created or updated are `v1` and contain only `v1` features.
```

Comment:
```suggestion
For risk-averse environments, we have added a **stable validation policy**, a [validating admission policy](https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/) which can ensures that only `v1` APIs and fields are used with Istio APIs.

In new environments, selecting the stable validation policy upon installing Istio will guarantee that all future Custom Resources created or updated are `v1` and contain only `v1` features.
```

LGTM overall!  Just a couple of fixes that were missed due to them being attached to questions and this is good to go from my end.

Thanks very much for writing this Whitney!